project/task1/handleWindowPoints.py

```diff
@@ -0,0 +1,12 @@
+def handleWindowPoints(windowPoints): 
```

Comment:
a moving average algorithm is much more efficient in this case.

Reply:
Will consider.